src/shared/components/alert-button/AlertButton.scss

```diff
@@ -0,0 +1,16 @@
+@import 'src/styles/common';
+
+.alertButton {
+  display: inline-block;
+  position: relative;
+  top: 4px;
```

Comment:
I really doubt `position: relative;  top: 4px;` should belong in this component. If it is to be reusable, it shouldn't know anything about the context in which it is put.

Reply:
So if i understand this right and looking at how question button works, we wrap the component within a div which can then be styled where its being used. if thats case i have moved the style out including the display as well.

Reply:
Done